GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml

```diff
@@ -30,6 +30,8 @@
 <span class="govuk-caption-xl">Administration</span>
 <h1 class="govuk-heading-xl">Database integrity checks</h1>
```

Comment:
Do we still want to keep this header?

Reply:
Good question, I'm not sure, Jacob only said he'd like us to add subheadings is this something we'd want to check?

Reply:
Yes, can you please check with Jacob?

Reply:
Should I reach him via slack or via the tickets comment section?

Reply:
Jacob's happy with the Heading currently, so all good

Reply:
Great, thanks for checking!